internal/cmd/version_command.go

```diff
@@ -11,15 +11,26 @@ import (
 
 type VersionOutput struct {
 	Version string `json:"version"`
+
+	BuildGoVersion string `json:"go_version,omitempty"`
+	BuildGoOS      string `json:"go_os,omitempty"`
+	BuildGoArch    string `json:"go_arch,omitempty"`
```

Comment:
Nitpick: I know it doesn't affect the output, but would you mind omitting the `Build` prefix from the fields here? I think the purpose of each variable is clear without it.

Reply:
Also do you mind adding one more field

```go
Compiler string `json:"compiler,omitempty"`
```
which would use `runtime.Compiler` constant https://golang.org/pkg/runtime/#pkg-constants